### PR TITLE
Enable strict mode

### PR DESCRIPTION
**Pull request by BenElgar**

### CODE REVIEW

File: gui/src/lib/process/store.ts [hunk 1]
```diff
@@ -35,26 +36,29 @@ export const fetchProcesses = (workspace) => {
       case 'success':
         refetchProcesses(workspace);
         return refetchingResponse(req.data);
+      case 'refetching':
+        // I'm not sure this is right.
+        return req;
```
Comment (BenElgar) [inline gui/src/lib/process/store.ts:41]: @kendru is this the correct thing to do here?

File: gui/src/lib/api.ts [hunk 2]
```diff
@@ -151,6 +151,30 @@ export interface WorkspaceDescription {
   root: string;
 }
 
+export interface WorkspaceApi {
```
Comment (brandonbloom) [inline gui/src/lib/api.ts:154]: we should extend our code generator to produce these too :)

File: gui/src/lib/process/store.ts (hunk 1 above)
Comment (brandonbloom) [reply gui/src/lib/process/store.ts:41]: seems reasonable to me, but i think this whole remote data / fetching mechanism needs some love

**Review by brandonbloom — APPROVED** [review]
Thanks for doing this!

**Pull request merged**